firestore-send-email/functions/src/index.ts

```diff
@@ -77,32 +76,28 @@ function validateFieldArray(field: string, array?: string[]) {
   }
 }
 
-function getExpireAt(startTime: admin.firestore.Timestamp) {
```

Comment:
is there a reason this stuff is being removed in this PR? Seems like some mix up of commits?